lib/schemas/validate.js

```diff
@@ -26,57 +27,61 @@ const eikJSON = createValidator(eikJSONSchema, {
     useDefaults: true,
 });
 const createNameValidator = (jsonSchemaValidator) => (value) => {
-        const result = jsonSchemaValidator(value);
-        if (!result.error) {
-            const pkvalid = npmPkg(value);
-            const errors = [];
-            if (!pkvalid.validForNewPackages) {
-                errors.push({
-                    keyword: 'validForNewPackages',
-                    dataPath: '.name',
-                    schemaPath: '',
-                    params: [],
-                    message: 'should be valid for new packages'
-                });
-            }
-            if (!pkvalid.validForOldPackages) {
-                errors.push({
-                    keyword: 'validForOldPackages',
-                    dataPath: '.name',
-                    schemaPath: '',
-                    params: [],
-                    message: 'should be valid for old packages'
-                });
-            }
-            if (errors.length) {
-                result.error = errors;
-            }
+    const result = jsonSchemaValidator(value);
+    if (!result.error) {
+        const pkvalid = npmPkg(value);
+        const errors = [];
+        if (!pkvalid.validForNewPackages) {
+            errors.push({
+                keyword: 'validForNewPackages',
+                dataPath: '.name',
+                schemaPath: '',
+                params: [],
+                message: 'should be valid for new packages',
+            });
+        }
+        if (!pkvalid.validForOldPackages) {
+            errors.push({
+                keyword: 'validForOldPackages',
+                dataPath: '.name',
+                schemaPath: '',
+                params: [],
+                message: 'should be valid for old packages',
+            });
+        }
```

Comment:
Do we need to cater for "old" packages at this stage? The amount of projects using this is probably at a level where we can manually update them in an hour or so.

Reply:
I'm squinting at this code (it's only in this PR due to formatting changes) and I think I get what's going on. Seems like we should be able to remove the old validation bit.